install/mysql-install.sh

```diff
@@ -24,9 +24,32 @@ $STD apt-get install -y \
   mc
 msg_ok "Installed Dependencies"
 
+PS3="Please select a Major Release: "
+RELEASES=("8.0" "8.4 - LTS" "9.1 - Innovation")
+select RELEASE in "${RELEASES[@]}"; do
+  case ${RELEASE} in
+    "8.0")
+      RELEASE_REPO="mysql-8.0"
+      RELEASE_AUTH="mysql_native_password"
+      break
+      ;;
+    "8.4 - LTS")
+      RELEASE_REPO="mysql-8.4-lts"
+      RELEASE_AUTH="caching_sha2_password"
+      break
+      ;;
+    "9.1 - Innovation")
```

Comment:
Is Innovation a stable Release? The name looks like a prototype / beta

Reply:
_Both Innovation and LTS are production-grade quality. If you desire access to the latest features and improvements and enjoy staying on top of the latest technologies, the MySQL Innovation releases might be the best fit for you._ [Source](https://dev.mysql.com/blog-archive/introducing-mysql-innovation-and-long-term-support-lts-versions/)